README.md

```diff
@@ -3,15 +3,22 @@
   <img alt="Heroku Cloud Native Buildpacks Logo" src="./logo.png" width="100%">
 </p>
 
-## What are Heroku Cloud Native Buildpacks?
+Create a production-ready container image for your application with a [single command](TUTORIAL.md); no Dockerfile required. [Try Heroku Cloud Native Buildpacks today](TUTORIAL.md).
 
-Heroku Cloud Native Buildpacks, now in preview, can help you create a production ready container image for your app with a single command, without the need to write a Dockerfile or configuring a complex build system.
+## What is a Cloud Native Buildpack?
 
-Heroku Cloud Native Buildpacks implement the [Cloud Native Buildpacks specification](https://buildpacks.io/) (CNB) and are free and open-source software. They can be used independently of the Heroku platform.
+In 2012, Heroku introduced the world to the declarative build system now known as [classic buildpacks](https://blog.heroku.com/buildpacks). Twelve years later, we've joined forces with the Cloud Native Compute Foundation (CNCF) to bring the next evolution in building reliable and consistent images to everyone with [Cloud Native Buildpacks](https://buildpacks.io).
+
+Cloud Native Buildpacks (CNBs) are an [open specification for building and running container images](https://buildpacks.io/) by the CNCF. Heroku Cloud Native Buildpacks implement the CNB spec, are open-source, can be used independently of the Heroku platform, and are available now as a preview.
 
 ### Philosophy
 
-Heroku CNBs are written with the app developer in mind. We want it to be as easy and simple as possible to transform an app into an OCI image. Our buildpacks are highly opinionated and apply industry best-practices without requiring additional configuration.
+While a Dockerfile is procedural, CNBs are declarative. Give us your application, and we'll give you an image that can run it. Don't believe us? [Try it out today](TUTORIAL.md).
```

Comment:
> While a Dockerfile is procedural, CNBs are declarative

I feel like this calls for a (likely very interesting) in-depth exploration/definition of the philosophical differences between procedural and declarative approaches as applied in this context (and "with the app developer in mind"). Without that, I think there's a likelihood this rephrasing will lead to more confusion than clarity for someone just starting to learn about CNB.

Reply:
I tend to agree I think this section is in pretty good shape as is.

Reply:
ok. Thanks for the context!

Keeping in mind this is the "philosophy" section, so I'm consuming this as an ideal not a guarantee. Using terms like "easy" and "best-practices" are ok imo b/c it's a philosophy. And yes, people will judge if it's those things and then feed that back so we can improve towards our goal.

That being said your points are valid. Maybe just combine the best parts of each?

> Heroku CNBs are written with the app developer in mind. We want it to be as easy and simple as possible to transform an app into an OCI image. While a Dockerfile is procedural, CNBs are declarative. Give us your application, and we'll give you an image that can run it. Don't believe us? [Try it out today](TUTORIAL.md).

Reply:
I knew there was a very interesting explanation (and lots of thoughts) behind this change! ;-)

I also agree that "highly opinionated" and "industry best-practices" are used too much/without substantiating what that means.

I like [@jonnymacs's suggestion](https://github.com/heroku/buildpacks/pull/11#discussion_r1530844638) to combine them -- the first two lines explain our goal/mission, and _"While a Dockerfile is procedural, CNBs are declarative"_ is how we make that happen. The following three sentences underscore both the declarative approach, and encourages/challenges people to see if we've achieved our goals :)

Seems like a reasonable compromise?

Reply:
I took Jonny's change wholesale.